sharepoint/sharepoint-ps/sharepoint-online/Set-SPOApplicationPermission.md

```diff
@@ -0,0 +1,144 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/set-spoapplicationpermission
+applicable: SharePoint Online
+title: Set-SPOApplicationPermission
+schema: 2.0.0
+author: cindylay
+ms.author: cindylay
+ms.reviewer:
+---
+
+# Set-SPOApplicationPermission
+
+## SYNOPSIS
+
+Manages permissions for a guest application to access a SharePoint Embedded application.
+
+## SYNTAX
+
+
+### ParamSet1
+
+```powershell
+Set-SPOApplicationPermission 
+[[-OwningApplicationId] <OwningApplicationid>] 
+[[-ApplicationId] <ApplicationId>]
+[[-PermissionAppOnly] <AppOnlyPermission>]
+[[-PermissionDelegated] <DelegatedPermission>]
+``` 
+
+## DESCRIPTION
+
+The `Set-SPOApplicationPermission` cmdlet manages permissions for a guest app's access to a SharePoint Embedded application. This includes adding, updating, and deleting guest app permissions. Currently only app-only permissions are supported at this time. Delegated permissions are not supported for guest applications. A guest application is defined as any application within the enterprise applications of the owning tenant. 
```

Comment:
>app [](http://example.com/codeflow?start=74&length=3)

is the terminology to use "app" or "application"?

(here and elsewhere)

Reply:
changed to application